src/memory.rs

```diff
@@ -507,21 +523,29 @@ impl UndefMask {
 
     /// Check whether the range `start..end` (end-exclusive) is entirely defined.
     fn is_range_defined(&self, start: usize, end: usize) -> bool {
-        if end > self.len { return false; }
+        if end > self.len {
+            return false;
+        }
```

Comment:
This and the follow few diffs seem to make the code bigger for no great readability improvement.